Rakefile

```diff
@@ -5,6 +5,16 @@ task :build do
   sh "bundle exec jekyll build"
 end
 
+desc "Serve the website"
+task :serve do
+  sh "bundle exec jekyll serve"
+end
+
+desc "Cleanup the website"
+task :clean do
+  sh "bundle exec jekyll clean"
```

Comment:
it basically just deletes the contents of the destination directory

http://www.rubydoc.info/github/mojombo/jekyll/Jekyll/Commands/Clean